spec/models/book_spec.rb

```diff
@@ -28,39 +27,57 @@
   let(:book) { create(:book) }
   let(:user) { create(:user) }
 
-  it '有効ファクトリーの確認' do
-    expect(book).to be_valid
-  end
-
-  context 'search' do
+  context "search" do
     let(:book) { create(:book, :book_with_comment) }
     subject { Book.find_with_comments(book.id) }
-    it 'success' do
+    it "success" do
       expect(subject.comments.size).to eq(3)
     end
   end
 
-  context 'has_favorites?' do
+  context "search" do
+    let(:search_param) { nil }
+    subject { Book.search(search_param) }
+
+    context 'without search_param' do
+      it "all books" do
+        expect(subject.size).to eq(Book.all.size)
+      end
+    end
+
+    context 'wit search_param' do
+      let(:search_param) { @timestamp }
+      before {
+        timestamp!
+        create(:book, body: "<h1>Title#{@timestamp}</h1>")
```

Comment:
ここのせいです。下記のようにすればいいですが削除するので関係ないですね。
```suggestion
        create(:book)
```